sdk/cosmos/azure-cosmos/README.md

```diff
@@ -581,6 +581,26 @@ def integrated_cache_snippet():
 ```
 For more information on Integrated Cache, see [Azure Cosmos DB integrated cache - Overview][cosmos_integrated_cache].
 
+### Configuring network transport
+
+ CosmosDB SDK under the hood is using `requests` library as a network transport. In case you would like to change it's settings you may pass it into `CosmosClient` constructor or `from_connection_string` class method.
+
+ For example if you would like to alter connection pool you can initialise `RequestsTransport` with an instance of `requests.Session`.
```

Comment:
Thank you so much @annatisch for reviewing my PR. Cool, I hope I have done a good job applying your comments. 
Do you think this is a good sync client example, or should I add something more?
Do you want me to add async example as well here? or maybe add something to `/sample` dir?

This questions is a bit longer but since SDK gives a lot of goodies for free, why isn't there an easier/safer way to modify the connection pool? Reason I'm going through all of this is that 10 connections is not enough for my use case. I'd like to change only that but keep all the good stuff. Is it a wrong approach I have taken? or is it the only approach?